Rubberduck.UnitTesting/UnitTesting/TestCodeGeneratorStatics.cs

```diff
@@ -0,0 +1,140 @@
+        private static string TestErrorMessage => $"{TestExplorer.UnitTest_NewMethod_RaisedTestError}: #";
+        private static string ExpectedErrorConstant => TestExplorer.UnitTest_NewMethod_ExpectedError;
+        private static string ExpectedErrorTodoComment => $"'TODO {TestExplorer.UnitTest_NewMethod_ChangeErrorNo}";
+        private static string ExpectedErrorFailMessage => TestExplorer.UnitTest_NewMethod_ErrorNotRaised;
+
+        private static string AccessCompareOption => $"Option Compare Database{Environment.NewLine}";
+        private static string LateBindConstName => TestExplorer.UnitTest_NewModule_LateBindConstant;
+        private static string DefaultTestFolder => TestExplorer.UnitTest_NewModule_DefaultFolder;
+        private static string ModuleInitializeMethod => TestExplorer.UnitTest_NewMethod_ModuleInitializeMethod;
+        private static string ModuleInitializeComment => $"'{TestExplorer.UnitTest_NewModule_RunOnce}.";
+        private static string ModuleCleanupMethod => TestExplorer.UnitTest_NewMethod_ModuleCleanupMethod;
+        private static string ModuleCleanupComment => $"'{TestExplorer.UnitTest_NewModule_RunOnce}.";
+        private static string TestInitializeMethod => TestExplorer.UnitTest_NewMethod_TestInitializeMethod;
+        private static string TestInitializeComment => $"'{TestExplorer.UnitTest_NewModule_RunBeforeTest}.";
+        private static string TestCleanupMethod => TestExplorer.UnitTest_NewMethod_TestCleanupMethod;
+        private static string TestCleanupComment => $"'{TestExplorer.UnitTest_NewModule_RunAfterTest}.";
+
+        private static string TestMethodTemplate =>
+$@"'@TestMethod(""{DefaultTestCategory}"")
+Public Sub {{0}}() {RenameTestTodoComment}
+    On Error GoTo {TestFailLabel}
+    
+    {ArrangeComment}
+
+    {ActComment}
+
+    '{AssertLabel}:
+    Assert.Succeed
+
+{TestExitLabel}:
+    Exit Sub
+{TestFailLabel}:
+    Assert.Fail ""{TestErrorMessage}"" & Err.Number & "" - "" & Err.Description
+End Sub";
+
+        private static string TestMethodExpectedErrorTemplate =>
+$@"'@TestMethod(""{DefaultTestCategory}"")
+Public Sub {{0}}() {RenameTestTodoComment}
+    Const {ExpectedErrorConstant} As Long = 0 {ExpectedErrorTodoComment}
+    On Error GoTo {TestFailLabel}
+    
+    {ArrangeComment}
+
+    {ActComment}
+
+{AssertLabel}:
+    Assert.Fail ""{ExpectedErrorFailMessage}""
+
+{TestExitLabel}:
+    Exit Sub
+{TestFailLabel}:
+    If Err.Number = {ExpectedErrorConstant} Then
+        Resume {TestExitLabel}
+    Else
+        Resume {AssertLabel}
+    End If
+End Sub";
+
+        private static string LateBindingDeclarations =>
+@"    Private Assert As Object
+    Private Fakes As Object";
+
+        private static string EarlyBindingDeclarations =>
+@"    Private Assert As Rubberduck.{0}
+    Private Fakes As Rubberduck.FakesProvider";
+
+        private static string DualBindingDeclarations =>
+$@"#Const {LateBindConstName} = 0
```

Comment:
The only potential issue that I see with that is that it could potentially leave the project in a state where it couldn't be compiled after adding a test module, and we always do a reparse on a module add. I do need to verify that the RD reference gets added as needed when this template is used though.